library/components/TTable.vue

```diff
@@ -0,0 +1,1018 @@
+<template>
+  <div>
+    <!-- The TColumnConfig that ends up in the slot is an unrecognized component, the back end must be adding them based on what is used in the page
+    so add this here just to tell the back end that this component uses the TColumnConfig component -->
+    <TColumnConfig :columnConfig="['test']"></TColumnConfig>
+    <slot name="columnConfig" :setColumnConfig="setColumnConfig"></slot>
+
+    <TSearch
+      v-if="can_search"
+      ref="easySearch"
+      @updateSearchTerm="updateSearchTerm"
+      :highlight-query-selector="
+        enableSearchHighlight ? '#tableContainer' : null
+      "
+      :highlight-options="highlightOptions"
+    ></TSearch>
+
+    <div
+      v-if="isDataAvailable"
+      id="tableContainer"
+      ref="tableContainer"
+      :style="columnWidthsStyle"
+    >
+      <!-- Title -->
+      <div v-if="title !== null" id="title" class="spanAllColumns center_cell">
```

Comment:
```suggestion
      <div v-if="title" id="title" class="spanAllColumns center_cell">
```
This should be enough.

Reply:
👍